### PR TITLE
Enable tests related to Helm 3.2 bin.

### PR DESCRIPTION
**Pull request by pratikjagrut**
Enable shells tests for Helm 3.2 which were disabled for Helm 3.1.

### CODE REVIEW

File: scripts/completion-tests/completionTests.sh [hunk 1]
```diff
@@ -620,4 +593,4 @@ unalias helm
 
 # This must be the last call.  It allows to exit with an exit code
 # that reflects the final status of all the tests.
-_completionTests_exit
+_completionTests_exit
```
Comment (pmacik) [inline scripts/completion-tests/completionTests.sh]: Could you please keep the end-of-file new-line in place? Thanks

**Review by pmacik — CHANGES_REQUESTED** [review]
Other than the end-of-file new-line, LGTM.
Verified with helm v3.2.1

File: scripts/completion-tests/completionTests.sh (hunk 1 above)
Comment (pratikjagrut) [reply scripts/completion-tests/completionTests.sh]: Ok

Comment (pratikjagrut) [reply scripts/completion-tests/completionTests.sh]: Done

Comment (pmacik) [reply scripts/completion-tests/completionTests.sh]: Thx

**Review by pmacik — APPROVED** [review]
LGTM

**Review by pmacik — CHANGES_REQUESTED** [review]
@pratikjagrut Please squash the commits and put a nice commit message explaining the change including the sign-off.

**Pull request merged**